### PR TITLE
Demo imageprovider docs

### PR DESCRIPTION
**Pull request by klucsik**
Docs for https://github.com/open-telemetry/opentelemetry-demo/pull/1462

### CODE REVIEW

File: content/en/docs/demo/services/imageprovider.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Imageprovider Service
+linkTitle: Imageprovider
+aliases: [imageprovider]
+---
+
+This service provides the images whcih are used in the frontend. Thei mages are statically hosted on a nginx instance. The nginx server is insturemnted with [nginx-otel module](https://github.com/nginxinc/nginx-otel/tree/main)
```
Comment (svrnm) [inline content/en/docs/demo/services/imageprovider.md:7]:
```suggestion
This service provides the images which are used in the frontend. The images are statically hosted on a nginx instance. The nginx server is instrumented with the [nginx-otel module](https://github.com/nginxinc/nginx-otel/tree/main).
```

**Review by svrnm — CHANGES_REQUESTED** [review]

**Review by mviitane — APPROVED** [review]

File: content/en/docs/demo/services/imageprovider.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: Imageprovider Service
```
Comment (svrnm) [inline content/en/docs/demo/services/imageprovider.md:2]:
```suggestion
title: Image Provider Service
```

File: content/en/docs/demo/services/imageprovider.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+linkTitle: Imageprovider
```
Comment (svrnm) [inline content/en/docs/demo/services/imageprovider.md:3]:
```suggestion
linkTitle: Image Provider
```

File: content/en/docs/demo/services/imageprovider.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+aliases: [imageprovider]
```
Comment (svrnm) [inline content/en/docs/demo/services/imageprovider.md:4]:
```suggestion
```

File: content/en/docs/demo/services/imageprovider.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+---
+
+This service provides the images which are used in the frontend. The images are statically hosted on a nginx instance. The nginx server is instrumented with the [nginx-otel module](https://github.com/nginxinc/nginx-otel/tree/main).
```
Comment (svrnm) [inline content/en/docs/demo/services/imageprovider.md:7]:
```suggestion
This service provides the images which are used in the frontend. The images are statically hosted on a NGINX instance. The NGINX server is instrumented with the [nginx-otel module](https://github.com/nginxinc/nginx-otel/tree/main).
```

**Review by svrnm — CHANGES_REQUESTED** [review]

**Review by theletterf — APPROVED** [review]

**Review by svrnm — APPROVED** [review]

**Comment by svrnm** [comment]
thank you @klucsik

**Comment by abirfo** [comment]
c83c48330022839de92f0b2a580379a50c0f43e6

**Pull request merged**